plugins/faustwp/includes/settings/functions.php

```diff
@@ -29,6 +29,15 @@ function is_rewrites_enabled() {
 	return '1' === faustwp_get_setting( 'enable_rewrites' );
 }
 
+/**
+ * Determines if posts and category URLs should link to the WP site.
+ *
+ * @return bool
+ */
+function use_wp_domain_for_post_and_category_urls() {
```

Comment:
I introduced this function and `use_wp_domain_for_media` to make it easier to understand. Since Faust was rewriting content multiple times, the existing functions were a bit confusing IMO when reasoning about booleans. You had to know Faust rewrites multiple times to understand it.